x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichShardMultiSearchAction.java

```diff
@@ -233,7 +235,8 @@ protected MultiSearchResponse shardOperation(Request request, ShardId shardId) t
                     shardId.id(),
                     searcher,
                     () -> { throw new UnsupportedOperationException(); },
-                    null
+                    null,
+                    emptyMap()  // NOCOMMIT is it right not to use the runtime mappings?
```

Comment:
When I looked I think that this one gets a search source builder, so it seemed that runtime mappings can be specified, hence we should use them?